k8s/supportability/src/lib.rs

```diff
@@ -47,6 +47,10 @@ pub struct SupportArgs {
     /// Kubernetes namespace of mayastor service, defaults to mayastor
     #[clap(global = true, long, short = 'n', default_value = "mayastor")]
     namespace: String,
+
+    /// Collect current pod logs along with historical logs
+    #[clap(global = true, short, long)]
+    current_logs: bool,
```

Comment:
Is the default value "false"?

Reply:
Yes. If we don't pass this flag, it will be set as false.